src/installer/pkg/sfx/installers.proj

```diff
@@ -11,7 +11,7 @@
 
   <ItemGroup Condition="'$(BuildRpmPackage)' == 'true'">
     <InstallerProjectReference Include="installers/dotnet-runtime-deps/dotnet-runtime-deps-azl.3.proj" />
-    <InstallerProjectReference Include="installers/dotnet-runtime-deps/dotnet-runtime-deps-centos.8.proj" />
+    <InstallerProjectReference Include="installers/dotnet-runtime-deps/dotnet-runtime-deps-centos.9.proj" />
```

Comment:
I presume there's no need to build `centos-8` deps package anymore, is that correct?

Reply:
The only consumer of the deps package are our other packages, so we don't need the centos-8 deps package any more.